todo-list/client/src/components/SettingsTab/index.jsx

```diff
@@ -0,0 +1,10 @@
+import SettingsApplicationsSharpIcon from '@mui/icons-material/SettingsApplicationsSharp';
+import { Box } from '@mui/material';
+
+ export const SettingsTab = () =>{
+    return(
+       <Box>
+        <SettingsApplicationsSharpIcon />
+       </Box>
```

Comment:
I am not sure, that you need to use a special component just to add `<Box>`. I think we can remove this component and use `SettingsApplicationsSharpIcon` in the parent component, but wrap it in the `Box`